src/realm/table.cpp

```diff
@@ -2562,10 +2555,9 @@ TableView Table::find_all_binary(ColKey, BinaryData)
     throw util::runtime_error("Not implemented");
 }
 
-ConstTableView Table::find_all_binary(ColKey, BinaryData) const
+ConstTableView Table::find_all_binary(ColKey col_key, BinaryData value) const
 {
-    // FIXME: Implement this!
-    throw util::runtime_error("Not implemented");
+    return const_cast<Table*>(this)->find_all_binary(col_key, value);
```

Comment:
Looks like this actually add new features :-)

Reply:
But it doesn't :-)